tests/unit/socketAuthTest.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-class PusherSocketAuthTest extends PHPUnit\Framework\TestCase
+class socketAuthTest extends PHPUnit\Framework\TestCase
```

Comment:
PSR-1+4 says class names must start with an uppercase letter, and the filename must match the classname.